package bbts.tracker;

message TrackerConf {
  optional int32  port                         = 1;
  optional int32  thread_num                   = 2;
  optional string conf_dir                     = 3;
  optional string log_path                     = 4;
  optional int32  httpd_port                   = 5;
  optional int32  monitor_cycle_second         = 6;
  optional int32  info_hash_expire_time        = 7;
  optional int32  remote_peers_syncronizer_num = 8;
  optional int32  server_send_timeout          = 9;
  optional int32  server_recv_timeout          = 10;
  optional int32  server_accept_timeout        = 11;
  optional int32  task_expire_time             = 12;
}

